problem_builder/plot.py

```diff
@@ -0,0 +1,258 @@
+        display_name=_("Plot title"),
+        default="Plot",
+        scope=Scope.content
+    )
+
+    plot_label = String(
+        display_name=_("Plot label"),
+        help=_("Label for default overlay that shows student's answers to scale questions"),
+        default="yours",
+        scope=Scope.content
+    )
+
+    point_color_default = String(
+        display_name=_("Point color (default overlay)"),
+        help=_("Point color to use for default overlay"),
+        default="green",
+        scope=Scope.content
+    )
+
+    point_color_average = String(
+        display_name=_("Point color (average overlay)"),
+        help=_("Point color to use for average overlay"),
+        default="blue",
+        scope=Scope.content
+    )
+
+    q1_label = String(
+        display_name=_("Quadrant I"),
+        help=_(
+            "Label for the first quadrant. "
+            "Plot uses counter-clockwise numbering starting in the top right quadrant."
+        ),
+        default="Q1",
+        scope=Scope.content
+    )
+
+    q2_label = String(
+        display_name=_("Quadrant II"),
+        help=_(
+            "Label for the second quadrant. "
+            "Plot uses counter-clockwise numbering starting in the top right quadrant."
+        ),
+        default="Q2",
+        scope=Scope.content
+    )
+
+    q3_label = String(
+        display_name=_("Quadrant III"),
+        help=_(
+            "Label for the third quadrant. "
+            "Plot uses counter-clockwise numbering starting in the top right quadrant."
+        ),
+        default="Q3",
+        scope=Scope.content
+    )
+
+    q4_label = String(
+        display_name=_("Quadrant IV"),
+        help=_(
+            "Label for the fourth quadrant. "
+            "Plot uses counter-clockwise numbering starting in the top right quadrant."
+        ),
+        default="Q4",
+        scope=Scope.content
+    )
+
+    claims = String(
+        display_name=_("Claims and associated questions"),
+        help=_(
+            'Claims and questions that should be included in the plot. '
+            'Each line defines a triple of the form "claim, q1, q2", '
+            'where "claim" is arbitrary text that represents a claim, '
+            'and "q1" and "q2" are IDs of scale questions. '
+        ),
+        default="",
+        multiline_editor=True,
+        resettable_editor=False
+    )
+
+    editable_fields = (
+        'display_name', 'plot_label', 'point_color_default', 'point_color_average',
+        'q1_label', 'q2_label', 'q3_label', 'q4_label', 'claims'
+    )
+
+    @lazy
+    def course_key_str(self):
+        location = _normalize_id(self.location)
+        return unicode(location.course_key)
+
+    @property
+    def default_claims(self):
+        return self._get_claims(self._get_default_response)
+
+    @property
+    def average_claims(self):
+        return self._get_claims(self._get_average_response)
+
+    def _get_claims(self, response_function):
```

Comment:
You'd be able to eliminate this function or significantly reduce it by using children as mentioned in the earlier comment.